dbt/adapters/redshift/impl.py

```diff
@@ -9,6 +9,9 @@
 
 
 class RedshiftAdapter(PostgresAdapter):
+
+    DEFAULT_TCP_KEEPALIVE = 240
```

Comment:
just curious... where did this number come from?

Reply:
Drew linked me to [this](https://airflow.apache.org/_modules/postgres_hook.html) which has a docstring bit that says:
>Note: For Redshift, use keepalives_idle in the extra connection parameters and set it to less than 300 seconds.

I wasn't sure how precise the `300` value was or where they sourced it from, so I figured one minute less would give us a bit more wiggle room on slightly flaky connections and stuff. Now that I think about it, they probably did the same thing with whatever their source was and 300 is also conservative, but hey, a few extra keepalives shouldn't hurt.

Reply:
👍